sdk/maps/azure-maps-render/azure/maps/render/aio/_patch.py

```diff
@@ -35,18 +35,23 @@ def _authentication_policy(credential):
         raise ValueError("Parameter 'credential' must not be None.")
     if isinstance(credential, AzureKeyCredential):
         authentication_policy = AzureKeyCredentialPolicy(name="subscription-key", credential=credential)
+    elif isinstance(credential, AzureSasCredential):
+        authentication_policy = AzureSasCredentialPolicy(credential)
     elif credential is not None and not hasattr(credential, "get_token"):
         raise TypeError(
             "Unsupported credential: {}. Use an instance of AzureKeyCredential "
-            "or a token credential from azure.identity".format(type(credential))
+            "or AzureSasCredential  or a token credential from azure.identity".format(type(credential))
```

Comment:
There's a redundant white space before second `or` word. Please correct for other `_patch.py` files in other packages.